wandb/sdk/launch/agent2/controllers/local_process.py

```diff
@@ -65,28 +63,23 @@ def __init__(
         legacy: LegacyResources,
         max_concurrency: int,
     ):
-        self.config = config
-        self.logger = logger
-        self.legacy = legacy
-        self.max_concurrency = max_concurrency
-
-        self.id = config["jobset_spec"].name
-        self.active_runs: Dict[str, AbstractRun] = {}
-
-        self.queue_driver = passthrough.PassthroughQueueDriver(
-            api=jobset.api,
-            queue_name=config["jobset_spec"].name,
-            entity=config["jobset_spec"].entity_name,
-            project=config["jobset_spec"].project_name,
-            agent_id=config["agent_id"],
+        self.queue_driver: passthrough.PassthroughQueueDriver = (
+            passthrough.PassthroughQueueDriver(
+                api=jobset.api,
+                queue_name=config["jobset_spec"].name,
+                entity=config["jobset_spec"].entity_name,
+                project=config["jobset_spec"].project_name,
+                agent_id=config["agent_id"],
+            )
         )
+        super().__init__(config, jobset, logger, legacy, max_concurrency)
 
-    async def pop_next_item(self) -> Any:
+    async def pop_next_item(self) -> Optional[Job]:
         next_item = await self.queue_driver.pop_from_run_queue()
         self.logger.info(f" item: {json.dumps(next_item, indent=2)}")
         return next_item
```

Comment:
This method + reconcile can be removed since they're in the base class

Reply:
Reconcile is implemented a little differently here, I'm going to leave it for now